app/main.py

```diff
@@ -2,0 +1,52 @@
+class SoftwareEngineer:
+    def __init__(self, name: str) -> None:
+        self.name = name
+        self.skills = []
+
+    def learn_skill(self, skill: str) -> None:
+        self.skills.append(skill)
+
+
+class FrontendDeveloper(SoftwareEngineer):
+    def __init__(self, name: str) -> None:
+        super().__init__(name)
+        super().learn_skill("JavaScript")
+        super().learn_skill("HTML")
+        super().learn_skill("CSS")
+
+    def create_awesome_web_page(self) -> str:
+        print(f"{self.name} is creating a webpage...")
+        return "<h1>Hello world</h1>"
+
+
+class BackendDeveloper(SoftwareEngineer):
+    def __init__(self, name: str) -> None:
+        super().__init__(name)
+        super().learn_skill("Python")
+        super().learn_skill("SQL")
+        super().learn_skill("Django")
```

Comment:
In the `BackendDeveloper` constructor, replace the use of `super().learn_skill()` with the `extend()` method or `+=` operator to add multiple skills at once to the `skills` list, as per the checklist guidelines .